### ISSUE TITLE
Dashes

### ISSUE THREAD

**Issue by RMeli** [texworld/betterbib#245]
I encountered several references where instead of using `-` in the title the following Unicode characters were used:
* U+2013: `–`
* U+2212: `−`

The presence of such characters makes compilation fail. 

Should the `.bib` file be sanitized by transforming these two characters into `-` or `--`? (I think in titles the former is more appropriate)

**Comment by RMeli** [comment]
In #137 is seems that there was something on those lines
```
value = value.replace("\u2010", "-")
```
but it's been removed?

**Comment by RMeli** [comment]
I switched to a different document where I have been using `betterbib` for a while, added one reference, and run `betterbib`. Several instances of the problem described above started to appear (in references different to the one that was added).

Is it possible that something added/removed in the latest version is causing this problem? Or am I missing something obvious here?

**Comment by nschloe** [comment]
As usual, and MWE is needed.

**Comment by nschloe** [comment]
Can you try the latest version?

**Comment by nschloe** [comment]
Alright. This is about the dash in the title, right?

**Comment by RMeli** [comment]
Yes, that's what causing the problem now. I have sanitised this bibliography before using `betterbib` without issue. In #137 it seems that the following line of code have been removed:
```
value = value.replace("\u2010", "-")
```

In this example is `U+2013`, but I encountered the same issue with `U+2010` and `U+2212` as well.

**Comment by RMeli** [comment]
I just saw #239, maybe the same can be applied to other fields as well?

**Comment by nschloe** [comment]
This should now be fixed (4.3.6).

**Comment by RMeli** [comment]
Thank you @nschloe !
